docs/DEVELOPING.md

```diff
@@ -12,6 +12,138 @@
 - [PostgreSQL](https://www.elastic.co/guide/en/enterprise-search/master/connectors-postgresql.html)
 - [S3](https://www.elastic.co/guide/en/enterprise-search/master/connectors-s3.html)
 
+## Confluence Connector
+
+The [Elastic Confluence connector](../connectors/sources/confluence.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+### Usage
+
+To use this connector as a **connector client**, use the **build a connector** workflow. See [Connector clients and frameworks](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+### Compatibility
```

Comment:
We are missing some fundamental information on what Confluence object types are supported by this connector.  For example, does it automatically fetch all pages, spaces, blogs, comments, templates, labels, attachments, categories, etc..?

Reply:
This info should go in **Documents and syncs** subheading :)

Reply:
Done :)

Reply:
Commit id: [7f6b6a5](https://github.com/elastic/connectors-python/pull/647/commits/7f6b6a511a3a472a49f4b362da7f15af88e57d96)